login/routers/controllers/UserResponseServices.js

```diff
@@ -0,0 +1,22 @@
+const UserModel = require('../../db/models/UserModel.js');
+const CreateUserResponce = require('../../db/CreateUserResponce.js');
+const Sequelize = require('sequelize');
+
+const SequelizeOperators = Sequelize.Op;
+
+module.exports.SignIn = function SingIn(request, response) {
+  UserModel.User.findOne({
```

Comment:
I'd extract this functionality to user repository to abstract db code, read about the repository design pattern, it's very userful and common.

Reply:
I wanna make user reposiory but i dont know how
Will there be all user functional,routes etc. or only some files,like this and routes/middlewares calls